exercises/nucleotide-count/metadata.toml

```diff
@@ -1,3 +1,4 @@
+title = "nucleotide-count"
```

Comment:
This one is in the format of the soug rather than title case.

Reply:
Thank you, good catch!